File: level1/4 - Lecture de l'entrée/9 - Socles pour statues.c
//coded by cybereagle2001
#include <stdio.h>
#define repeat(nb) for (int _loop = 1, _max = (nb); _loop <= _max; _loop++)

int main()
{
   int largeurBas;
   int largeurHaut;
   scanf("%d", &largeurBas);
   scanf("%d", &largeurHaut);
   int volume = 0;
   int largeur = largeurHaut;
   repeat (largeurBas - largeurHaut + 1)
   {
      volume = volume + largeur * largeur;
      largeur = largeur + 1;
   }
   printf("%d\n", volume);
}

File: java/level 1/4-Lecture de l'entrée/9- Socles pour statues.java
/** coded by cybereagle2001 */

import algorea.Scanner;

class Main{
   static Scanner input= new Scanner(System.in);
   public static void main(String[] args){
      int x= input.nextInt();
      int y= input.nextInt();
      int z=0;
      for (int loop=0; loop<(x-y)+1; loop++){
         z=z+((x-loop)*(x-loop));
      }
      System.out.println(z);
   }
}
